lib/ecto_enum.ex

```diff
@@ -57,18 +64,34 @@ defmodule EctoEnum do
     end
   end
 
+  defmodule UndeterminableStorageError do
+    defexception [:message]
+
+    def exception(kw) do
+      msg = "cannot determine storage type for #{inspect kw}"
```

Comment:
Definitely.

I'm thinking something like:

> You have conflicting data types! EctoEnum can only store using one type (integers or strings but not both) for the same enum.

I can further extend the error message to indicate which member is the outlier:

> Specifically, `archived` is given as a string while other members are given as integers.